repos/nuget-client.proj

```diff
@@ -26,16 +26,32 @@
 
   <Target Name="RepoBuild"
           DependsOnTargets="InitSubmodules">
+    <PropertyGroup>
+      <BuildCommandBase>$(DotnetToolCommand) msbuild $(ProjectDirectory)/build/build.proj</BuildCommandBase>
+      <BuildCommandBase>$(BuildCommandBase) /p:VisualStudioVerion=15.0</BuildCommandBase>
+      <BuildCommandBase>$(BuildCommandBase) /p:Configuration=$(Configuration)</BuildCommandBase>
+      <BuildCommandBase>$(BuildCommandBase) /p:BuildRTM=false</BuildCommandBase>
+      <BuildCommandBase>$(BuildCommandBase) /p:BuildNumber=$(NuGetClientBuildNumber)</BuildCommandBase>
+    </PropertyGroup>
 
-    <Exec Command="$(DotnetToolCommand) msbuild $(ProjectDirectory)/build/build.proj /t:RestoreXPLAT /p:VisualStudioVerion=15.0 /p:Configuration=$(Configuration) /p:BuildRTM=false /p:BuildNumber=$(NuGetClientBuildNumber) /bl:restore.binlog $(RedirectRepoOutputToLog)"
+    <Exec Command="$(BuildCommandBase) /t:RestoreXPLAT /bl:restore.binlog $(RedirectRepoOutputToLog)"
           EnvironmentVariables="@(EnvironmentVariables)"
           WorkingDirectory="$(ProjectDirectory)" />
 
-    <Exec Command="$(DotnetToolCommand) msbuild $(ProjectDirectory)/build/build.proj /t:BuildXPLAT /p:VisualStudioVerion=15.0 /p:Configuration=$(Configuration) /p:BuildRTM=false /p:BuildNumber=$(NuGetClientBuildNumber) /bl:build.binlog $(RedirectRepoOutputToLog)"
+    <Exec Command="$(BuildCommandBase) /t:BuildXPLAT /bl:build.binlog $(RedirectRepoOutputToLog)"
           EnvironmentVariables="@(EnvironmentVariables)"
           WorkingDirectory="$(ProjectDirectory)" />
 
-    <Exec Command="$(DotnetToolCommand) msbuild $(ProjectDirectory)/build/build.proj /t:PackXPLAT /p:VisualStudioVerion=15.0 /p:Configuration=$(Configuration) /p:BuildRTM=false /p:BuildNumber=$(NuGetClientBuildNumber) /p:PackageOutputPath=$(PackagesOutput) /p:NoPackageAnalysis=true /flp:v=detailed /bl:pack.binlog $(RedirectRepoOutputToLog)"
+    <PropertyGroup>
+      <PackCommand>$(BuildCommandBase) /t:PackXPlat</PackCommand>
+      <PackCommand>$(PackCommand) /p:PackagOutputPath=$(PackagesOutput)</PackCommand>
```

Comment:
Typo: `PackagOutputPath` instead of `PackageOutputPath`.

Reply:
Nice spot, wonder how I managed that.

This is a weird property though, come to think of it... telling NuGet to put packages in `$(ProjectDirectory)artifacts/nupkgs`, but that's where NuGet would put them anyway. (Which is why the prebuilt baselines didn't change.) I'll just fix it for now, anyway--maybe it lets us be more flexible between NuGet submodule commits or something.